consensus/polybft/polybft_config.go

```diff
@@ -75,21 +77,40 @@ func (p *PolyBFTConfig) IsBridgeEnabled() bool {
 
 // Validator represents public information about validator accounts which are the part of genesis
 type Validator struct {
-	Address types.Address
-	BlsKey  string
-	Balance *big.Int
-	NodeID  string
+	Address       types.Address
+	BlsPrivateKey *bls.PrivateKey
+	BlsKey        string
+	BlsSignature  string
+	Balance       *big.Int
+	NodeID        string
 }
 
 type validatorRaw struct {
-	Address types.Address `json:"address"`
-	BlsKey  string        `json:"blsKey"`
-	Balance *string       `json:"balance"`
-	NodeID  string        `json:"nodeId"`
+	Address      types.Address `json:"address"`
+	BlsKey       string        `json:"blsKey"`
+	BlsSignature string        `json:"blsSignature"`
+	Balance      *string       `json:"balance"`
+	NodeID       string        `json:"nodeId"`
+}
+
+func (v *Validator) MakeKoskSignature(chainID int64) error {
```

Comment:
We already have a function with this name. Could we rename this to `InitKoskSignature`, since this function only changes state of the `Validator` object.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1184/commits/0683391e4b1b15ecfe0f2e80572d0444a795fbbe